extensions/wikia/PhalanxII/classes/PhalanxPager.class.php

```diff
@@ -172,7 +172,7 @@ function formatRow( $row ) {
 		if ( isset( $row->p_timestamp ) ) {
 			$html .= ' &bull; ';
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal " &bull; " 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)